dask/bag/core.py

```diff
@@ -62,6 +61,15 @@
     takes_multiple_arguments,
 )
 
+try:
+    from dask import multiprocessing
+except (ImportError, ModuleNotFoundError):
+    from dask import local
+
+    DEFAULT_GET = local.get_sync
+else:
+    DEFAULT_GET = multiprocessing.get
```

Comment:
I was surprised to learn that the default scheduler for `Bag` is multiprocessing, as opposed to multithreading for the other collections. Is this deliberate?

Reply:
Yeah, Bag tends to be a collection of regular Python objects, which tend to hold the GIL.

Reply:
Cool, thanks @TomAugspurger